<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login2</class>
 <widget class="QDockWidget" name="login2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>460</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255);</string>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="loginPage">
   <widget class="QPushButton" name="loginClose">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="userName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>320</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;border-radius:10px;border: 2px solid #e3e1ef;text-align : left;border-left-width : 2px;padding-top : 7px
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="passWord">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>320</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;border-radius:10px;border: 2px solid #e3e1ef;text-align : left;border-left-width : 2px;padding-top : 5px
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>320</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 53);border-radius:10px;border:1px;padding:1px 70px;
</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
